github/http.go

```diff
@@ -175,3 +172,45 @@ func proxyFromEnvironment(req *http.Request) (*url.URL, error) {
 
 	return proxyURL, nil
 }
+
+type simpleClient struct {
+	httpClient  *http.Client
+	rootUrl     *url.URL
+	accessToken string
+}
+
+func (c *simpleClient) Get(path string) (res *simpleResponse, err error) {
+	url, err := url.Parse(path)
+	if err != nil {
+		return
+	}
+
+	url = c.rootUrl.ResolveReference(url)
+	req, err := http.NewRequest("GET", url.String(), nil)
+	if err != nil {
+		return
+	}
+	req.Header.Set("Authorization", "token "+c.accessToken)
+
+	httpResponse, err := c.httpClient.Do(req)
+	if err == nil {
+		res = &simpleResponse{httpResponse}
+	}
+
+	return
+}
+
+type simpleResponse struct {
+	*http.Response
+}
+
+func (res *simpleResponse) Parse(dest interface{}) (err error) {
```

Comment:
I'm a bit indifference to this. But Go community seems to have the convention of naming deserialized method `Encode/Decode` or the object `Encoder/Decoder`, e.g., the JSON decoder etc. FWIW.